### ISSUE TITLE
read_search is not available in restapi resource

### ISSUE THREAD

**Issue by ogysha** [Mastercard/terraform-provider-restapi#91]
When I issue terraform plan/validate
with following resource:

```
resource "restapi_object" "de" {
  path         = var.api_path
  data         = file("${path.root}/data/languages/de.json")
  id_attribute = "items/iso2Code"
  read_path    = var.api_path
  read_search = {
    search_key   = "items/iso2Code"
    search_value = "de"
  }
}
```

I get error:
```
2020-07-31T13:05:19.936+0200 [DEBUG] plugin: plugin exited
Unsupported argument

  on modules\languages\de\main.tf line 6, in resource "restapi_object" "de":
   6:   read_search = {

An argument named "read_search" is not expected here.
```

I am using the latest v1.13.0 of the provider on Windows 10 x64.

**Comment by dl-mai** [comment]
Hi there is there any update on this? It does not seem to be released in v.13.0

**Comment by DRuggeri** [comment]
Oh, my gosh - you're right. This was accepted as a PR, but never released!

@fgouteroux - I'm embarrassed for missing this earlier - I've just released your PR #83 in `v1.14.0`
